bindata/network/infra-dpu-cni/infra-dpu-cni.yaml

```diff
@@ -0,0 +1,163 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: infra-dpu-cni-script
+  namespace: openshift-infra-dpu-cni
+  annotations:
+    kubernetes.io/description: |
+      This is a script that serves as the CNI on DPU. Only host network supported, so returns error to all calls.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+data:
+  infra-dpu-cni.sh: |-
+    #!/bin/bash
+    set -e
+
+    exec 3>&1 # make stdout available as fd 3 for the result
+
+    log=/var/log/infra-dpu-cni.log
+
+    cniVersionStr="0.4.0"
+    addErrorStr="Only host backed pods supported in Infra Cluster."
+    unknownErrorStr="Unknown cni command: $CNI_COMMAND"
+
+    echo >> $log
+    echo "COMMAND: $CNI_COMMAND" >> $log
+
+    case $CNI_COMMAND in
+    ADD)
+        echo ${addErrorStr} >> $log
+    echo "{
+      \"cniVersion\": \"${cniVersionStr}\",
+      \"code\": 7,
+      \"msg\": \"Not Supported\",
+      \"details\": \"${addErrorStr}\"
+    }" >&3
+        exit 1 
+    ;;
+
+    DEL)
+    ;;
+
+    GET)
+    ;;
+
+    VERSION)
+    echo "{
+      \"cniVersion\": \"${cniVersionStr}\", 
+      \"supportedVersions\": [ \"0.3.0\", \"0.3.1\", \"0.4.0\" ] 
+    }"
+    ;;
+
+    *)
+        echo ${unknownErrorStr} >> $log
+    echo "{
+      \"cniVersion\": \"${cniVersionStr}\",
+      \"code\": 4,
+      \"msg\": \"Invalid Configuration\",
+      \"details\": \"${unknownErrorStr}\"
+    }" >&3
+        exit 1 
+    ;;
+
+    esac
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: infra-dpu-cni-conf
+  namespace: openshift-infra-dpu-cni
+  annotations:
+    kubernetes.io/description: |
+      This is the Infra-DPU-CNI Readiness Indicator File to let Multus know CNI is ready.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+data:
+  08-infra-dpu-cni.conf: |-
+    {"cniVersion":"0.4.0","name":"infra-dpu-cni","type":"infra-dpu-cni"}
+
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: infra-dpu-cni-plugin
+  namespace: openshift-infra-dpu-cni
```

Comment:
Can this be squashed into the existing multus daemonset somehow? This is adding another set of pods that will be created, do something at startup, and then sit around consuming some resources and cluttering up `oc get` output forever after